### PR TITLE
Feature/add rest service file upload

### PR DESCRIPTION
**Pull request by ghost**
@samsamann, also please check the backend. The API now should work and upload the file to resources/uploads/ ...

### CODE REVIEW

File: src/main/resources/project-defaults.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ swarm:
     data-sources:
       defaultDB:
         user-name: "sa"
-        connection-url: "jdbc:h2:tcp://localhost:1521/test"
+        connection-url: "jdbc:h2:tcp://localhost:1818/test"
```
Comment (ghost) [inline src/main/resources/project-defaults.yml]: Attention port 1818 works for me, because 1521 doesn't work...

Comment (samsamann) [reply src/main/resources/project-defaults.yml]: let the config untouched. use another config file for your development.

File: src/main/java/com/github/bfh/study/slb/rest/FileUploadService.java [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+/**
+ * File Upload Service
+ *
+ * @author Dario Carosella
+ */
+@Path("file")
+public class FileUploadService {
+
+    private static final Logger _log = LoggerFactory.getLogger(FileUploadService.class);
```
Comment (samsamann) [inline src/main/java/com/github/bfh/study/slb/rest/FileUploadService.java]: do not use underlines. use uppercase

File: src/main/java/com/github/bfh/study/slb/rest/FileUploadService.java [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    private static final String SERVER_UPLOAD_LOCATION_FOLDER = "src/main/resources/uploads/";
```
Comment (samsamann) [inline src/main/java/com/github/bfh/study/slb/rest/FileUploadService.java]: move this to the config file.

Comment (ghost) [reply src/main/java/com/github/bfh/study/slb/rest/FileUploadService.java]: in which config file?

File: src/main/java/com/github/bfh/study/slb/rest/FileUploadService.java (hunk 2 above)
Comment (ghost) [reply src/main/java/com/github/bfh/study/slb/rest/FileUploadService.java]: uppercase for class variable are for Constants... but logger isn't a constant. So I corrected it to lowercase as you have in the class PrepareStep.java (line: 24)

**Review by samsamann — APPROVED** [review]

**Pull request merged**